nginx/huly.nginx

```diff
@@ -1,6 +1,8 @@
 server {
     listen 80;
     server_name _;
+    client_max_body_size 100M;
```

Comment:
```suggestion
    client_max_body_size          0;
```
Might be even better to lift client_max_body_size and control it through the outer reverse proxy or host configuration.

Reply:
I use nginx as reverse proxy - so there is Huly and nginx in front. Had problems with attaching files to issues in Huly and increased the client_max_body_size

Reply:
> Might be even better to lift client_max_body_size and control it through the outer reverse proxy or host configuration, what do you think?

Agree, if huly could control on own side (not checked yet, just started reading the docs)